src/dotnet-core-uninstall/Shared/Configs/CommandLineConfigs.cs

```diff
@@ -171,74 +167,57 @@ internal static class CommandLineConfigs
         };
 
         public static readonly ParseResult CommandLineParseResult;
-        public static readonly IEnumerable<Option> UninstallAuxOptions;
+        public static readonly IEnumerable<Option> RemoveAuxOptions;
+        public static readonly IEnumerable<Option> DryRunAuxOptions;
+        public static readonly IEnumerable<Option> WhatIfAuxOptions;
         public static readonly IEnumerable<Option> ListAuxOptions;
 
-        static CommandLineConfigs()
+        static CommandLineConfigs() 
         {
-            UninstallRootCommand.AddArgument(new Argument<IEnumerable<string>>
-            {
-                Name = LocalizableStrings.UninstallNoOptionArgumentName,
-                Description = LocalizableStrings.UninstallNoOptionArgumentDescription
-            });
-
             UninstallRootCommand.AddCommand(ListCommand);
+            UninstallRootCommand.AddCommand(DryRunCommand);
```

Comment:
I find system.commandline has build-in alias for command. 

Related to the comment below for string vs Option. We do need to use this field later. So it does not need to be static then, make it readonly and initialize in constructor. 

```c#
// put in constructor
            this.DryRunCommand = new Command(
               DryRunCommandName,
               LocalizableStrings.DryRunCommandDescription);

            DryRunCommand.AddAlias("whatif");

// put in constructor end
            UninstallRootCommand.AddCommand(DryRunCommand);
```